git-keeper-client/gkeepclient/fetch_submissions.py

```diff
@@ -160,23 +161,6 @@ def pull_repo_if_updated(local_repo_path, remote_url, remote_head_hash,
         print(str(e))
 
 
-def build_clone_url(path):
-    """
-    Build a git clone URL with the following form:
-
-    ssh://<username>@<hostname>:<port number>/<path>
-
-    The username, hostname, and port number come from the configuration file,
-    which needs to be parsed.
-
-    :param path: path to the repository on the server
-    :return:
-    """
-    return ('ssh://{0}@{1}:{2}/{3}'
-            .format(config.server_username, config.server_host,
-                    config.server_ssh_port, path))
-
-
```

Comment:
This appears to remove all the documentation on the function.  Should there be something here?

Reply:
Other than my one comment about the code, this all looks good.

Knowing it's run in production for so long helps!